src/avatar.jsx

```diff
@@ -156,7 +154,7 @@ const Avatar = React.createClass({
           className={className}
         >
           {icon && React.cloneElement(icon, {
-            color: color,
+            color: color || avatar.color,
```

Comment:
What do you think about add a color property to `styles.icon` and setting it to `color || avatar.color` there and then putting `styles.icon.color` in the cloneElement call.

This just centralizes the styling logic for the component.

As a general thought I'm not a huge fan of the color props on the API. I would prefer if we could steer users to controlling color through normal style/css because it's more natural I think and should help simplify code on our end.